src/index.html

```diff
@@ -17,16 +17,24 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <div class="search-wrapper">
+      <form class="search-bar search-bar--big" data-qa="big">
+        <input
+          class="search-bar__input--big"
+          type="text"
+          data-qa="keypress"
+          placeholder="Try “Los Angeles“"
+        />
+      </form>
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+      <form class="search-bar search-bar--small" data-qa="small">
+        <input
+          class="search-bar__input--small"
+          type="text"
+          data-qa="hover"
+          placeholder="Try “Los Angeles“"
+        />
```

Comment:
The form elements lack the `action` and `method` attributes. Even if they're not going to be used in this case, it's a good practice to include them for a complete form definition.